horton/scripts/wpart.py

```diff
@@ -17,9 +17,8 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, see <http://www.gnu.org/licenses/>
 #
-#--
-'''Utility functions for the ``horton-wpart.py`` script'''
-
+# --
```

Comment:
Can you undo this change? This type of errors should all be fixed at once, together with a fix in the script `updateheaders.sh`. See #17. This pep8 error should not be causing problems.